packages/apps/plugins/plugin-table/src/components/ObjectTable/hooks/useObjects.ts

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { useFilteredObjects } from '@braneframe/plugin-search';
+import { type EchoReactiveObject, Filter, type S } from '@dxos/echo-schema';
+import { type Space, useQuery } from '@dxos/react-client/echo';
+
+// TODO(Zan): Consolidate.
+const Stable = {
+  empty: {
+    object: Object.freeze({}),
+    array: Object.freeze([] as any[]),
+  },
+};
+
+export const useObjects = (space?: Space, schema?: S.Schema<any>) => {
```

Comment:
could you explain why this is needed?

Reply:
It's just a little noisy in the code since it composes the query and filtering together.

Reply:
The object table shouldn't care about filtering, and it felt weird seeing 'filtered objects' in the rest of the component implementation.

Reply:
Could you maybe add a jsdoc for these hooks to explain intended usage?